lib/Subscriber/Behavior/Path/AutoNameSubscriber.php

```diff
@@ -196,6 +196,9 @@ private function getName(AutoNameBehavior $document, NodeInterface $parentNode,
 
         $name = $this->slugifier->slugify($title);
 
+        // jackrabbit can not handle node-names which contains a number followed by "e" e.g. 10e
+        $name = preg_replace('((\d+)e)', '$1-e', $name);
```

Comment:
i think in my analysis at https://github.com/jackalope/jackalope/issues/313 i found that only .-% before a number trigger the problem. and that uppercase E is also a problem. maybe `'([\.-%]\d+)([eE])', '$1-$2` ?